master/networking/determine-best-networking.md

```diff
@@ -0,0 +1,110 @@
+| Google cloud                                                 | Google GKE                                   | Excellent                                                | Low                  |                                                              |
+| Flannel                                                      | Any public cloud                             | Poor to excellent, depending on chosen backend           | Medium               | Does not support full {{site.prodname}} IPAM feature set                |
+
+
+### Tutorial
+
+This section provides more details on {{site.prodname}}’s built-in networking options: 
+
+- [Unencapsulated, peered with physical infrastructure](#unencapsulated-peered-with-physical-infrastructure)
+- [Unencapsulated, not peered with physical infrastructure](#unencapsulated-not-peered-with-physical-infrastructure)
+- [IP in IP or VXLAN encapsulation](#ip-in-ip-or-vxlan-encapsulation)
+
+#### Unencapsulated, peered with physical infrastructure
+
+{{site.prodname}} can peer with your routers using BGP. This provides great performance and easy debugging of unencapsulated traffic, and a wide range of options for network topology and connectivity. On top of the advantages of non-peered unencapsulated traffic:
+
+- Your cluster can span multiple L2 subnets without needing encapsulation
+- Resources outside your cluster can talk directly to your pods without NAT
+- You can even expose pods directly to the internet if you want!
+
+To configure BGP peering and determine the right topology, see [Configure BGP peering]({{site.baseurl}}/{{page.version}}/networking/bgp). This option requires the ability to configure BGP peers on your routers.  If this is not an option, see the next section.
+
+#### Unencapsulated, not peered with physical infrastructure
+
+This option also provides near host-to-host levels of performance and allows the network direct visibility of traffic.
+
+{{site.prodname}} can route pod traffic between nodes without encapsulation when all nodes are on a single L2 subnet, and if the underlying network doesn’t enforce IP address checks.  If your network consists of multiple L2 subnets then you can either [peer over BGP with your routers]({{site.baseurl}}/{{page.version}}/networking/bgp), or use [cross-subnet encapsulation](#ip-in-ip-or-vxlan-encapsulation) to encapsulate only traffic that crosses subnet boundaries.
+  
+Traffic cannot be routed between pods and destinations that aren’t also in the {{site.prodname}} cluster without allowing workload access outside cluster, or peering with infrastructure.
+
+> **Tip**: On AWS, you can disable source/destination checking to use this option within a VPC subnet. [Can I run Calico in a public cloud environment?]({{site.baseurl}}/{{page.version}}/reference/faq#can-i-run-calico-in-a-public-cloud-environment).
+{: .alert .alert-info}
+
+#### IP in IP or VXLAN encapsulation
```

Comment:
We should probably link to this document somewhere in this section: https://docs.projectcalico.org/v3.10/networking/vxlan-ipip

Reply:
Added.